### ISSUE TITLE
Incorrect handling of lists as selectors

### ISSUE THREAD

**Issue by Limych** [leafo/scssphp#244]
SCSS code:
```
@function tester() {
	@return (foo, bar);
}
.test	{
	#{tester()} {
		border: 1px dashed red;
	}
}
```

Expected:
```
.test foo, .test bar {
  border: 1px dashed red; }
```

Actual:
```
.test foo, bar {
  border: 1px dashed red; }
```

**Comment by vita10gy** [comment]
Is this related to #248? Seems like the same problem, reversed.

**Comment by Limych** [comment]
Yes, it's same problem.

**Comment by vita10gy** [comment]
I wish I had any idea where to even start looking to fix it. Assuming the general approach is multiple passes (maybe not a good assumption, but it's a common compiler approach), an early pass would need to make it so it looked like the user just typed it out as

```scss
.test   {
   foo{
        border: 1px dashed red;
    }
   bar{
        border: 1px dashed red;
    }
}
```

to subsequent passes.

**Comment by robocoder** [comment]
Fixed in 1409169
